common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -672,6 +672,27 @@ private Credential getFamilyRefreshTokenForAccount(@NonNull final AccountRecord
         return result;
     }
 
+    /**
+     * Load  FRTs from the cache for an account matching the homeAccountId
+     * @param homeAccountId : homeAccountId for which FRT is sought
+     * @return an FRT if available else null.
+     */
+    @Nullable
+    public RefreshTokenRecord getFamilyRefreshTokenForHomeAccountId(@NonNull final String homeAccountId) {
```

Comment:
Can I get you to add two tests to the `MsalOAuth2TokenCacheTest` for this? 1 to test successful retrieval and another to test `null` when no matches

Reply:
added tests as suggested